Common/Securities/SecurityPortfolioManager.cs

```diff
@@ -398,14 +398,18 @@ public decimal TotalPortfolioValue
                     {
                         var position = kvp.Value;
                         var securityType = position.Type;
-                        // we can't include forex in this calculation since we would be double accounting with respect to the cash book
-                        // we also exclude futures and CFD as they are calculated separately
+                        // We can't include forex in this calculation since we would be double accounting with respect to the cash book
+                        // We also exclude futures and CFD as they are calculated separately because they do not impact the account's cash.
+                        // We include futures options as part of this calculation because IB chooses to change our account's cash balance
```

Comment:
uf. any idea why?

Reply:
I believe CME says that futures options are cash settled, but call/put writing requires margin and has a initial/maintenance margin